build-system/git.js

```diff
@@ -120,13 +126,12 @@ exports.gitDiffColor = function() {
 };
 
 /**
- * Returns the name of the branch from which the PR originated. On Travis, this
- * is exposed via TRAVIS_PULL_REQUEST_BRANCH.
+ * Returns the name of the branch from which the PR originated.
  * @return {string}
  */
 exports.gitBranchName = function() {
-  return process.env.TRAVIS ?
-    process.env.TRAVIS_PULL_REQUEST_BRANCH :
+  return isTravisPullRequestBuild() ?
```

Comment:
This doesn't look like it, but it's a logic change - my intuition says "this is fine", my other intuition says "this is how things end up exploding in our face".

While this seems more correct ("if Travis pull request then return Travis pull request's branch" compared with "if Travis then return Travis pull request's branch"), are you 1,000% certain that we're not using `gitBranchName()` incorrectly somewhere and accept an empty return value?